platform/ios/src/MGLMapView.mm

```diff
@@ -4631,6 +4646,9 @@ - (void)updateAnnotationViews
 
     if (!delegateImplementsViewForAnnotation)
     {
+        [CATransaction begin];
+        [self updateCalloutView];
```

Comment:
Have you considered moving the calls to `-updateCalloutView` to directly within the `MapChangeDidFinishRenderingFrame` handler, after the call to `-updateAnnotationViews`? That would avoid the double call here, and I think it’d be more intuitive, since we need to reposition the callout view even if there are no view-backed annotations.

For reference, the macOS implementation calls `-updateAnnotationCallouts` within the `MapChangeRegionIsChanging` and `MapChangeRegionDidChange` handlers, which may fire less frequently. It makes sense that we’d want to keep the callout view in sync with any annotation views we update in `MapChangeDidFinishRenderingFrame`, though.